Topics/Development_Process/Code_Smells.md

```diff
@@ -0,0 +1,45 @@
+## Code Smells
+Code smells refer to certain types of code that, while functional, will require increasing accommodation as more code begins to rely on the "smelly" code. While it is possible to ignore these types of code,
+the longer that they remain, the harder it becomes to fix issues they directly or indirectly cause in the future. Therefore, it's in a developer's best interest to be familiar with a broad spectrum of code smells
+so that they may identify and eliminate them as soon as possible.
```

Comment:
Newline should be removed from here since it unnecessarily makes the code look different from the preview.

Reply:
Good catch, this has been fixed now